packages/api/src/index.js

```diff
@@ -124,7 +124,9 @@ r.add(
 // Upload
 r.add('get', '/check/:cid', withMode(nftCheck, RO), [postCors])
 r.add('get', '', withAuth(withMode(nftList, RO)), [postCors])
-r.add('get', '/:cid', withAuth(withMode(nftGet, RO)), [postCors])
+r.add('get', /^\/(?<cid>ba\S+|Qm\S+)/i, withAuth(withMode(nftGet, RO)), [
```

Comment:
What if we added:

```js
r.add('get', '/login', () => new Response({ status: 405, statusText: 'Method Not Allowed' }), [postCors])
r.add('get', '/:cid', withAuth(withMode(nftGet, RO)), [postCors])
```

* No regex
* Allows other multibases to be supported if we want